modules/wmi/charts.go

```diff
@@ -598,6 +611,132 @@ var (
 	}
 )
 
+// IIS
+var (
+	IISCharts = module.Charts{
+		IISActiveConnectionChartsChart.Copy(),
+		IISActiveRequestChart.Copy(),
+		IISIsapiExtReqChart.Copy(),
+		IISUptimeChart.Copy(),
+		IISBandwidthChart.Copy(),
+		IISTotalRequestsChart.Copy(),
+		IISConnsAttempChart.Copy(),
+		IISRequestAttempChart.Copy(),
+		IISFileTransferChart.Copy(),
+		IISExtensionReqChart.Copy(),
+		IISLogonChart.Copy(),
+		IISErrorChart.Copy(),
+	}
+	IISActiveConnectionChartsChart = module.Chart{
+		ID:       "iis_active_conn",
+		Title:    "Website Active connections",
+		Units:    "connections",
+		Fam:      "IIS",
+		Ctx:      "wmi.iis_active_conn",
+		Type:     module.Stacked,
+		Priority: prioIISActiveConns,
```

Comment:
We need a chart per site, not a dimension per site.

Reply:
Done